Jenkinsfile

```diff
@@ -0,0 +1,50 @@
+pipeline {
+    agent {
+        node {
+            label 'centos7-docker-4c-2g'
+        }
+    }
+    options {
+        timestamps()
+        timeout(360)
+    }
+    stages {   
+        stage('Lint YAML files') {
+            steps {
+                script {
+                    def yamllintImage = "nexus3.edgexfoundry.org:10003/edgex-devops/yamllint:latest"
+
+                    docker.image(yamllintImage).inside("-v ${env.WORKSPACE}:/ws -w /ws --entrypoint=''") {
```

Comment:
You shouldn't have to bind mount the workspace to /ws. By default when you use docker.image.inside it automatically mount the workspace in the image.